provisioning/containers/gui-common/README.md

```diff
@@ -0,0 +1,22 @@
+# Docker-based GUI
+
+This sub-project aims to provide a Dockerized environment where Docker-based GUI applications can run. This is backed by a set of different containers, each with its own specific functions, that together constitute and offer the Docker-based GUI application.
+
+## General architecture
+
+In order to deploy GUI applications in CrownLabs the following is required:
+
+- A VNC-based X server
+- A window manager (it's better to have it since sub-windows or menu opened by the application might not get rendered correctly)
+- Websockify
+- noVNC client files (static HTML files hosted somewhere)
+- the GUI application
+
```

Comment:
Maybe not here, but you have to document how those components are installed and run. Everything in the same Docker? Or in multiple Dockers? (e.g., sidecars)?